_data/locales/en.yml

```diff
@@ -28,6 +28,10 @@ subtitle_project: openSUSE Project
 addnotice: To have your blog added to this aggregator, please read $instructions
 addnotice_instructions: the instructions
 
+alt_text_avatar_prefix: the avatar of
+alt_text_avatar_default: a silhouette of a person's head and shoulders, used as a default avatar
+alt_text_planet_logo: OpenSUSE's Geeko chameleon's head overlayed on a cell-shaded planet Earth, rotated to show the continents of Europe and Africa
```

Comment:
openSUSE (same as in https://github.com/openSUSE/planet-o-o/pull/78#discussion_r1527203146)

Reply:
Ah, does this apply to the first line of the sentence as well?